sretoolbox/binaries/oc.py

```diff
@@ -0,0 +1,72 @@
+"""
+Abstractions around the OC binary.
+"""
+
+import os
+
+import tarfile
+
+from semver import VersionInfo
+
+from sretoolbox.binaries.base import Binary
+
+
+class Oc(Binary):
+    """
+    Defines the properties of OC.
+    """
+    binary = 'oc'
+    download_path = '/tmp'
+    download_url = ('https://mirror.openshift.com/pub/'
+                    'openshift-v{major}/'
+                    'clients/ocp/'
+                    '{major}.{minor}.{patch}/'
+                    'openshift-client-linux-{major}.{minor}.{patch}.tar.gz')
+
+    def get_version_command(self):
+        """
+        Gets the command and its option(s) to check the version.
+
+        :return: version command
+        :rtype: list
+        """
+        return [self.command, 'version', '--client']
+
+    @staticmethod
+    def parse_version(version):
+        """
+        Parses version string as returned by the command execution
+        to a VersionInfo instance.
+
+        :param version: the return from the version command
+        :type version: str
+
+        :return: the parsed version as a VersionInfo object
+        :rtype: VersionInfo
+        """
+        # Example:
+        # Client Version: 4.6.1
+        opm_version = version.split(':')[1].strip()
+        return VersionInfo.parse(version=opm_version)
```

Comment:
this should probably be oc_version for correctness

Reply:
fixed